ql/src/java/org/apache/hadoop/hive/ql/lockmgr/DbTxnManager.java

```diff
@@ -485,6 +496,26 @@ private void clearLocksAndHB() {
     stopHeartbeat();
   }
 
+  private void cleanupDirForCTAS() throws LockException {
```

Comment:
I would make the method name more generic, like `cleanupResultDir`/ `cleanupOutputDir`, so it could be extended in the future.

Reply:
Done